globals.go

```diff
@@ -163,6 +167,8 @@ var (
 	disableSampling = new(int32)
 )
 
+type exitFunc func(int)
```

Comment:
Please see here why I suggest to remove this

https://github.com/rs/zerolog/pull/717#discussion_r2006416883

```suggestion
```

Also having an non-exported type on ExitFunc that is q globally exported func is a bad thing